app/controllers/quotes_controller.rb

```diff
@@ -32,4 +32,18 @@ def destroy
     quote = Quote.find_by_id!(params[:id])
     delete_object(quote)
   end
+
+  def anonymous_error(text)
+    flash.now[:error] = text
+    render turbo_stream: turbo_stream.update('flash', partial: 'layouts/alert')
+  end
+
+  def anonymous
```

Comment:
En de actienaam beschrijft niet wat er gebeurd en het kan beter een aparte controller zijn of de standaard quote update route gebruiken.